server/routes/shared_api/observation/series.py

```diff
@@ -64,6 +65,22 @@ def series_within():
   variables = list(filter(lambda x: x != "", request.args.getlist('variables')))
   if not variables:
     return 'error: must provide a `variables` field', 400
+  batch_size = request.args.get('batch_size') or 5000
+  # Make batched calls if it's a special case, as defined in shared.py
+  if (child_type in shared.NEEDS_SPECIAL_HANDLING and
+      parent_entity in shared.NEEDS_SPECIAL_HANDLING[child_type]):
+    try:
+      child_places = fetch.descendent_places([parent_entity],
```

Comment:
nit: use `.get(parent_entity, [])` instead of just accessing it in case the key isn't in the response?

Reply:
Good point, switched to using .get()